below/dump/src/system.rs

```diff
@@ -37,12 +37,29 @@ impl Dumper for System {
         round: &mut usize,
         comma_flag: bool,
     ) -> Result<IterExecResult> {
+        let mut fields = self.fields.clone();
+
+        if self.opts.detail || self.opts.everything {
+            // If detail is set, add per-cpu fields.
+            for key in model.system.cpus.keys() {
+                for subquery_id in &enum_iterator::all::<model::SingleCpuModelFieldId>().collect::<Vec<_>>() {
+                    let value = subquery_id.clone();
+                    fields.push(DumpField::FieldId(model::SystemModelFieldId::Cpus(
+                        model::BTreeMapFieldId {
+                            key: Some(*key),
+                            subquery_id: value,
+                        },
+                    )));
+                }
+            }
```

Comment:
IIUC, this is necessary b/c we cannot statically know how many CPUs are in the model. So we must hydrate fields we want to query at runtime, and per dataframe (b/c cpus can go offline).

If that's right, would be good to document that in a comment